app/javascript/packs/application.js

```diff
@@ -2,11 +2,14 @@
 // present in this directory. You're encouraged to place your actual application logic in
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
-
+require("jquery")
 // require("@rails/ujs").start()
 require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
+require("custom/pagination")
+
+
 
 
```

Comment:
why to make a separate folder for javascript ?? In rails there's app/assets/javascript .. learn abt the asset pipeline..Why we place all js file to that folder only ??

Reply:
As advised I have gone through the asset pipeline and adding new JavaScript into this. 
I am currently using Rails 6  and in this version, JavaScript assets are no longer included with the app/assets/ directory. They have been moved into a separate directory app/javascript. And one of the ways to add a new JS file is to create a custom directory and require it in application.js ( I have followed this method). But initially, I was confused with this so I have created a new javascript folder in the assets but I now realized that it is not needed.